controllers/registry.go

```diff
@@ -816,9 +871,9 @@ func (r *DPAReconciler) ReconcileRegistrySVCs(log logr.Logger) (bool, error) {
 	bslList := velerov1.BackupStorageLocationList{}
 	if err := r.List(r.Context, &bslList, &client.ListOptions{
 		Namespace: r.NamespacedName.Namespace,
-		LabelSelector: labels.SelectorFromSet(map[string]string{
-			"app.kubernetes.io/component": "bsl",
-		}),
+		// LabelSelector: labels.SelectorFromSet(map[string]string{
```

Comment:
Why comment this out?

Reply:
we want to enable registry for BSLs created outside DPA as well.